site/en/docs/lighthouse/accessibility/scoring/index.md

```diff
@@ -34,179 +34,231 @@ aren't included in the table because they don't affect your score.
+        <td>7</td>
+      </tr>
+      <tr>
+        <td><a href="https://dequeuniversity.com/rules/axe/4.7/meta-refresh">The document does not use <code>&#60;meta http-equiv="refresh"&#62;</code></a></td>
         <td>10</td>
       </tr>
       <tr>
-        <td><a href="https://dequeuniversity.com/rules/axe/4.4/meta-viewport"><code>[user-scalable="no"]</code> is not used in the <code>&#60;meta name="viewport"&#62;</code> element and the <code>[maximum-scale]</code> attribute is not less than 5.</a></td>
+        <td><a href="https://dequeuniversity.com/rules/axe/4.7/meta-viewport"><code>[user-scalable="no"]</code> is not used in the <code>&#60;meta name="viewport"&#62;</code> element and the <code>[maximum-scale]</code> attribute is not less than 5.</a></td>
         <td>10</td>
       </tr>
       <tr>
-        <td><a href="https://dequeuniversity.com/rules/axe/4.4/object-alt"><code>&#60;object&#62;</code> elements have alternate text</a></td>
-        <td>3</td>
+        <td><a href="https://dequeuniversity.com/rules/axe/4.7/object-alt"><code>&#60;object&#62;</code> elements have alternate text</a></td>
+        <td>7</td>
       </tr>
       <tr>
-        <td><a href="https://dequeuniversity.com/rules/axe/4.4/tabindex">No element has a <code>[tabindex]</code> value greater than 0</a></td>
-        <td>3</td>
+        <td><a href="https://dequeuniversity.com/rules/axe/4.7/select-name">Select elements have associated label elements.</a></td>
+        <td>7</td>
       </tr>
       <tr>
-        <td><a href="https://dequeuniversity.com/rules/axe/4.4/td-headers-attr">Cells in a <code>&#60;table&#62;</code> element that use the <code>[headers]</code> attribute refer to table cells within the same table.</a></td>
+        <td><a href="https://dequeuniversity.com/rules/axe/4.7/skip-link">Skip links are focusable</a></td>
         <td>3</td>
       </tr>
       <tr>
-        <td><a href="https://dequeuniversity.com/rules/axe/4.4/th-has-data-cells"><code>&#60;th&#62;</code> elements and elements with <code>[role="columnheader"/"rowheader"]</code> have data cells they describe.</a></td>
-        <td>3</td>
+        <td><a href="https://dequeuniversity.com/rules/axe/4.7/tabindex">No element has a <code>[tabindex]</code> value greater than 0</a></td>
+        <td>7</td>
       </tr>
       <tr>
-        <td><a href="https://dequeuniversity.com/rules/axe/4.4/valid-lang"><code>[lang]</code> attributes have a valid value</a></td>
-        <td>3</td>
+        <td><a href="https://dequeuniversity.com/rules/axe/4.7/table-duplicate-name">Tables have different content in the summary attribute and <code>&#60;caption&#62;</code>.</a></td>
+        <td>1</td>
+      </tr>
+      <tr>
+        <td><a href="https://dequeuniversity.com/rules/axe/4.7/table-fake-caption">Tables use <code>&#60;caption&#62;</code> instead of cells with the <code>[colspan]</code> attribute to indicate a caption.</a></td>
```

Comment:
```suggestion
        <td><a href="https://dequeuniversity.com/rules/axe/4.7/table-fake-caption">Tables use <code>&#60;caption&#62;</code> instead of cells with the <code>[colspan]</code> attribute to indicate a caption</a></td>
```